crates/re_log_types/src/time_point/non_min_i64.rs

```diff
@@ -0,0 +1,195 @@
+// Adapted from <https://github.com/LPGhatguy/nonmax>.
```

Comment:
Why no use the `nonmax` crate directly?

Reply:
I really want to reserve `i64::MIN` rather than `i64::MAX` for a variety of reasons, but `nonmax` doesn't have `NonMin` variants :sob:

Reply:
ah, makes sense. Maybe add a comment about that here then :)